### DISCUSSION TITLE
Errore Zeroconf

### DISCUSSION THREAD

**Discussion by Matteol82** [indomus/forum#351] [HUB: Home Assistant]
Ciao a tutti, ho un problema dopo l'aggiornamento alla versione 2021.5
All'avvio HA mi segnala un _"Error during setup of component zeroconf, [WinError 10049] Indirizzo richiesto non valido nel proprio contesto"_, il quale se non erro a leggere il log mi impedisce anche la partenza di:

- Unable to set up dependencies of sonoff. Setup failed for dependencies: zeroconf
- Unable to set up dependencies of default_config. Setup failed for dependencies: zeroconf

Leggendo un pò su internet mi sembra di aver capito che la versione 0.29 di zeroconf(che è quella contenuta in HA) soffriva di un problema su windows(io sono su windows) infatti ho visto che esiste già la versione 0.31

La mia domanda è, primo, se è già successo a qualcuno, perchè questa è tutta una mia deduzione, secondo, come faccio ad aggiornare quel singolo componente in HA.
Io ho provato a fare dalla linea dei comandi _pip install zeroconf==0.31.0_ ma mi sembra di aver visto che ha aggiornato solo le librerie globali di Python e non quelle dentro a HA.

Come potete capire sono ignorante in materia di Python.

Grazie a tutti

**Comment by Matteol82** [comment]
Nessuno riesce a darmi un aiutino?

**Reply by Newreader** [reply]
Prova a leggere qui:
https://community.home-assistant.io/t/zeroconf-error/153883/4

**Reply by Matteol82** [reply]
Grazie mille effettivamente è bastato inserire nel file di configurazione:

```
zeroconf:
  default_interface: true
```

**Reply by Newreader** [reply]
Bene
